modules/cluster/main.tf

```diff
@@ -23,13 +23,70 @@ resource "azurerm_kubernetes_cluster" "aks" {
 
   role_based_access_control {
     enabled = true
+  }
+}
 
-    azure_active_directory {
```

Comment:
Not sure I understand why this is being removed as it is required to enable RBAC integration to AKS? Isn't this tangential to pods running under their own service principals?